crates/core/src/node/op_state_manager.rs

```diff
@@ -201,9 +201,9 @@ impl OpManager {
     /// Notify the operation manager that a transaction is being transacted over the network.
     pub fn sending_transaction(&self, peer: &PeerId, msg: &NetMessage) {
         let transaction = msg.id();
-        if let Some(loc) = msg.requested_location() {
+        if let (Some(recipient), Some(target)) = (msg.target(), msg.requested_location()) {
```

Comment:
an other way of doing this is: `if let Some((recipient, target)) = msg.target().zip(msg.requested_location()) ...`

Reply:
Yes, although I'm not sure using `zip` is more readable - unless there is an efficiency advantage.